.github/workflows/publish.yml

```
@@ -0,0 +1,20 @@
+name: Publish nightly
+on:
+  schedule:
+    - cron: '0 7 * * *'
```

```suggestion
    - cron: '0 5 * * *'
```

LR has a 5am thing. how about this also